orm/model/ormdb/module_test.go

```diff
@@ -276,7 +276,7 @@ func TestHooks(t *testing.T) {
 	db, err := ormdb.NewModuleDB(TestBankSchema, ormdb.ModuleDBOptions{})
 	assert.NilError(t, err)
 	hooks := ormmocks.NewMockHooks(ctrl)
-	ctx := ormtable.WrapContextDefault(ormtest.NewMemoryBackend().WithHooks(hooks))
```

Comment:
could we have a test for both types of hooks?

Reply:
updated this. turns out the mocks needed to be generated